code/modules/mob/living/carbon/xenomorph/abilities/queen/queen_abilities.dm

```diff
@@ -50,11 +50,11 @@
 	action_icon_state = "secrete_resin"
 	ability_name = "projected resin"
 	plasma_cost = 100
-	xeno_cooldown = 2 SECONDS
+	xeno_cooldown = 4 SECONDS
```

Comment:
Shouldn't this remain 2 seconds so that when build boost ends it then is 4 seconds?

Reply:
Build boost overrides to 0 cooldown slightly below, haven't changed that.
Didn't check if I can just set this to 0 and remove override without breaking something, but I think it'd hurt readability so would rather not.